draft-irtf-cfrg-frost.md

```diff
@@ -1128,6 +1127,29 @@ If `vss_verify` fails, the participant MUST abort the protocol, and failure shou
       return 0
 ~~~
 
+We now define how the coordinator and signing participants can derive group info, which is an input into the FROST signing protocol.
+
+~~~
+    derive_group_info(n, t, vss_commitment):
```

Comment:
```suggestion
    derive_group_info(MAX_SIGNERS, THRESHOLD_LIMIT, vss_commitment):
```

We now use MAX_SIGNERS and THRESHOLD_LIMIT instead of n and t, respectively. (Side bar: maybe we should rename THRESHOLD_LIMIT to MIN_SIGNERS?)

Reply:
I would prefer keeping threshold in the name, to keep in line with existing terminology.